### PR TITLE
[gpuCI] Forward-merge branch-21.12 to branch-22.02 [skip gpuci]

### PR DESCRIPTION
**Pull request by GPUtester**
Forward-merge triggered by push to `branch-21.12` that creates a PR to keep `branch-22.02` up-to-date. If this PR is unable to be immediately merged due to conflicts, it will remain open for the team to manually merge.

### CODE REVIEW

**Comment by GPUtester** [comment]
**FAILURE** - Unable to forward-merge due to conflicts, **manual** merge is necessary. Do not use the `Resolve conflicts` option in this PR, follow these instructions https://docs.rapids.ai/maintainers/gpuci/#forward-mergers 
 **IMPORTANT:** Before merging and approving this PR, be sure to change the merging strategy to `Create a Merge Commit`. Otherwise, history will be lost and the branches become incompatible.

**Review by ttnghia — APPROVED** [review]

**Review by karthikeyann — APPROVED** [review]

**Review by bdice — APPROVED** [review]
This is needed to unblock CI style checks for clang-format ([sample failure](https://gpuci.gpuopenanalytics.com/blue/organizations/jenkins/rapidsai%2Fgpuci%2Fcudf%2Fprb%2Fcudf-style/detail/cudf-style/21480/pipeline/)).

**Pull request merged**
